docs/advanced_topics/add_to_django_project.rst

```diff
@@ -83,34 +83,37 @@ Wagtail requires several Django app modules, third-party apps, and defines sever
 Wagtail Apps
 ------------
 
-``wagtailcore``
+``wagtail.core``
   The core functionality of Wagtail, such as the ``Page`` class, the Wagtail tree, and model fields.
 
-``wagtailadmin``
+``wagtail.admin``
   The administration interface for Wagtail, including page edit handlers.
 
-``wagtaildocs``
+``wagtail.documents``
   The Wagtail document content type.
 
-``wagtailsnippets``
+``wagtail.snippets``
   Editing interface for non-Page models and objects. See :ref:`Snippets`.
 
-``wagtailusers``
+``wagtail.users``
   User editing interface.
 
-``wagtailimages``
+``wagtail.images``
   The Wagtail image content type.
 
-``wagtailembeds``
+``wagtail.embeds``
   Module governing oEmbed and Embedly content in Wagtail rich text fields. See :ref:`inserting_videos`.
 
-``wagtailsearch``
+``wagtail.search``
   Search framework for Page content. See :ref:`wagtailsearch`.
 
-``wagtailredirects``
+``wagtail.sites``
+  The Wagtail site content type.
```

Comment:
The content type is actually in Wagtail core. This app just provides a management UI

Reply:
Oh, right. I just thought it would make sense to include it in the list, since it was in `INSTALLED_APPS`. We could change the text to "Management UI for Wagtail sites"?

Reply:
Yep, that sounds good to me, thanks!